restock-shop.lic

```diff
@@ -121,39 +118,26 @@ class ShopRestock
       end
     end
 
-    while bput('get burlap cloth from back', 'You get', 'What were') == 'You get'
+    while bput("get burlap cloth from #{@bag}", 'You get', 'What were') == 'You get'
       fput('combine cloth')
     end
     stow_hands
 
     missing_items.each do |item|
       move('out') if Room.current.id == @inner_id
-
-      item[6].times do
-        order_item(16_667, 11)
-        fput('stow padding')
+      recipe_data = recipe_lookup(@recipes, item['recipe'])
+      recipe_data['part'] << 'large padding' # HACK: Reinforcing should be user specified
```

Comment:
❓ What is this doing? I thought `large padding` was specified as part of the recipe data

Reply:
Large padding is being added to all recipes since all tailoring products are reinforced. This is the original behavior of the script

Reply:
Sorry, all burlap armor is reinforced

Reply:
Making this a user setting, `reinforce: true` would be a nice addition in a future update. In which case this hack can be refactored

Reply:
👍